jenkinsapi/jenkins.py

```diff
@@ -268,6 +269,9 @@ def get_queue_url(self):
         url = "%s/%s" % (self.base_server_url(), 'queue')
         return url
 
+    def get_use_baseurl(self):
```

Comment:
How about:
```
@property
def base_url(self):
    return self.use_baseurl
```
Since this is not doing any logic and just returning a property of the class, it would be appropriate to have this decorated a property.

Reply:
Thanks for the update... Done